ui/lib/core/addon/components/form-field-groups-loop.hbs

```diff
@@ -4,7 +4,12 @@
       {{#each fields as |attr|}}
```

Comment:
moved form-field-groups-loop into the addon folder and passed a yield through if there was one.